src/sage/matrix/strassen.pyx

```diff
@@ -312,6 +313,7 @@ def strassen_echelon(MatrixWindow A, cutoff):
     strassen_echelon_c(A, cutoff, A._matrix._strassen_default_cutoff(A._matrix))
     sig_off()
 
+
 cdef strassen_echelon_c(MatrixWindow A, Py_ssize_t cutoff, Py_ssize_t mul_cutoff):
     # The following notation will be used in the comments below, which should be understood to give
```

Comment:
outch, this is not nice...

Reply:
j'ai pas compris, quel est le souci ?

Reply:
normalement, les commentaires d'une fonction commencent pas ` ``` `, pas par `# ...`. Je ne sais pas si c'est un problème. Je note que ce n'est pas le standard de sagemath.

Reply:
ok. Ici, ce premier commentaire fait reference à d'autres commentaires plus loin dans le code. Je propose de laisser comme ca.